packages/core/src/common/_customs.scss

```diff
@@ -78,10 +81,9 @@ $dark-button-box-shadow-active: $custom-shadow;
 $tooltip-padding-vertical: 0;
 $tooltip-padding-horizontal: 0;
 $pt-tooltip-box-shadow: $custom-shadow;
+$dark-tooltip-background-color: $dark-gray0 !important;
```

Comment:
I gess there is no other way arround and you need that `!important` in a variable... but it is pretty uncommon.

Just want to double check here.